MerlinAU.sh

```diff
@@ -1272,6 +1279,8 @@ _CreateEMailContent_()
            emailBodyTitle="WARNING"
            {
              echo "Found high-risk phrases in the change-logs while Auto-Updating to version <b>${fwNewUpdateVersion}</b> on the <b>${MODEL_ID}</b> router."
+             echo "Changelog contents include the following changes:"
```

Comment:
I decided to go with both methods, have it in the email, and have it available on demand :)
Why not.